src/swaps/blindex/BlindexSwapProvider.js

```diff
@@ -0,0 +1,509 @@
+import BN from 'bignumber.js'
+import { v4 as uuidv4 } from 'uuid'
+import JSBI from 'jsbi'
+import * as ethers from 'ethers'
+
+import UniswapV2Router from '@uniswap/v2-periphery/build/IUniswapV2Router02.json'
+import { Token, CurrencyAmount, Fraction, Percent } from '@uniswap/sdk-core'
+import ERC20 from '@uniswap/v2-core/build/ERC20.json'
+import { ChainId, chains, currencyToUnit, unitToCurrency } from '@liquality/cryptoassets'
+import cryptoassets from '@/utils/cryptoassets'
+import { ChainNetworks } from '@/utils/networks'
+
+import { isERC20 } from '../../utils/asset'
+import { prettyBalance } from '../../utils/coinFormatter'
+import { withInterval, withLock } from '../../store/actions/performNextAction/utils'
+import { SwapProvider } from '../SwapProvider'
+import { BestRouteService } from './BestRouteService'
+import { blindexConfig } from './BlindexConfig'
+
+const SWAP_DEADLINE = 30 * 60 // 30 minutes
+const DECIMALS = 18
+const RSK_RPC_URL = 'https://public-node.rsk.co'
+
+class BlindexSwapProvider extends SwapProvider {
+  constructor(config) {
+    super(config)
+    this._apiCache = {}
+    this.bestRouteService = new BestRouteService()
+    this.wrbtcAddress = blindexConfig.ERC20_INFO.find((x) => x.symbol === 'WRBTC').address
+    this.nativeTokenSymbol = chains[ChainId.Rootstock].nativeAsset //RBTC
+    this.availableTokenSymbols = blindexConfig.ERC20_INFO.filter((entry) => entry !== 'UNI-V2').map(
```

Comment:
Once you'll remove the UNI tokens, this could be removed
```suggestion
    this.availableTokenSymbols = blindexConfig.ERC20_INFO.map(
```

Reply:
done